observability/logging/logging-6.0/log6x-upgrading-to-6.adoc

```diff
@@ -0,0 +1,434 @@
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+   - name: application-logs
+     type: application
+     application:
+       includes:
+       - namespace: foo
+       - namespace: bar
+       - namespace: my-important
+         container: main
+       excludes:
+       - container: too-verbose
+----
+
+[NOTE]
+====
+*application*, *infrastructure*, and *audit* are reserved words and cannot be used as names when defining an input.
+====
+
+=== Input Receivers
+
+Changes to input receivers include:
+
+* Explicit configuration of the type at the receiver level.
+* Port settings moved to the receiver level.
+
+.5.9 Input Receivers
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+  - name: an-http
+    receiver:
+      http:
+        port: 8443
+        format: kubeAPIAudit
+  - name: a-syslog
+    receiver:
+      type: syslog
+      syslog:
+        port: 9442
+----
+
+.6.0 Input Receivers
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+  - name: an-http
+    type: receiver
+    receiver:
+      type: http
+      port: 8443
+      http:
+        format: kubeAPIAudit
+  - name: a-syslog
+    type: receiver
+    receiver:
+      type: syslog
+      port: 9442
+----
+
+== Output Specifications
+
+High-level changes to output specifications include:
+
+* URL settings moved to each output type specification.
+* Tuning parameters moved to each output type specification.
+* Separation of TLS configuration from authentication.
+* Explicit configuration of keys and secret/configmap for TLS and authentication.
+
+== Secrets and TLS Configuration
+
+Secrets and TLS configurations are now separated into authentication and TLS configuration for each output. They must be explicitly defined in the specification rather than relying on administrators to define secrets with recognized keys. Upgrading TLS and authorization configurations requires administrators to understand previously recognized keys to continue using existing secrets. Examples in the following sections provide details on how to configure *ClusterLogForwarder* secrets to forward to existing Red Hat managed log storage solutions.
+
+== Red Hat Managed Elasticsearch
+
+.v5.9 Forwarding to Red Hat Managed Elasticsearch
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  logStore:
+    type: elasticsearch
+----
+
+.v6.0 Forwarding to Red Hat Managed Elasticsearch
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+  - name: default-elasticsearch
+    type: elasticsearch
+    elasticsearch:
+      url: https://elasticsearch:9200
+      version: 6
+      index: <log_type>-write-{+yyyy.MM.dd}
```

Comment:
This is not how the index logic works.    It should use a field path within curly brackets, and a fallback... to find `log_type`.    I'm not sure the {+yyyy.MM.dd} does what you think.... (maybe it does, but we haven't documented anything like it).   

https://docs.openshift.com/container-platform/4.16/observability/logging/logging-6.0/log6x-upgrading-to-6.html

> The Index can be a combination of static and dynamic values consisting of field paths followed by `||` followed by another field path or a static value.  A dynamic value is encased in single curly brackets `{}` and MUST end with a static fallback value separated with `||`.

Example from upstream docs:  
`index: '{.log_type||"nologformat"}-write'`

Reply:
I see this was commented on previously by several others.   Is there an existing bug to fix, or do we still need to create one?

Reply:
@libander, have you made a bug to track the remainder of the feedback on this PR?

Reply:
https://issues.redhat.com/browse/OBSDOCS-1345 for resolving these comments to existing content covered by this PR.  
https://issues.redhat.com/browse/OBSDOCS-1337 for tracking additions to the 6.0 docs.